code/index.html

```diff
@@ -1,32 +1,32 @@
 <!DOCTYPE html>
 <html lang="en">
-
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <link rel="stylesheet" href="./style.css" />
     <link
       href="https://fonts.googleapis.com/css2?family=Montserrat:ital,wght@0,100;0,200;0,300;0,400;0,500;0,600;0,700;0,800;0,900;1,100;1,200;1,300;1,400;1,500;1,600;1,700;1,800;1,900&display=swap"
-      rel="stylesheet" />
+      rel="stylesheet"
+    />
     <title>Chatbot</title>
   </head>
 
   <body>
-    <h1>Welcome to my chatbot!</h1>
+    <h1>Paws & Hooves Haven chatbot</h1>
     <main>
-      <section class="chat" id="chat"></section>
+      <section class="chat" id="chat">
+        <!--Här hamnar chatmeddelanden-->
```

Comment:
It's OK for learning purposes, but for future reference: comments should also be in English so that everybody (teachers, peers, recruiters, colleagues etc) understands.